2016-06-28-Rust-Reach-Fractal.md

```diff
@@ -0,0 +1,101 @@
+
+> **Device design should focus on user experience rather than on implementation.** Tessel uses high level languages and modular plug & play hardware (see philosophy here) so that device creators can fully prototype and test device functionality before worrying about optimization and implementation specifics.
+
+We kept these values with Tessel 2, and began efforts to help developers scale. This pushes toward another core belief:
+
+> **Tessel should be practical to use.** Tessel as a platform cannot have a big impact unless it is cost effective, reliable, and available for purchase. *It needs to be possible to purchase Tessel hardware at affordable prices, from quantity one up to production-level quantities (10,000+).*
+
+(Emphasis added.)
+
+## Where we are: the vision of Tessel 2
+
+When we designed Tessel 2, we envisioned it as the basis of full-scale products. At the core of its design was the key idea:
+
+**People should be able to scale the tools they use for prototyping all the way into production.**
+
+This is a radical idea.
+
+If you're a Node developer, this prototyping --> production path feels familiar. Design is an iterative process; you understand technology and needs better as you build. Modularity lets you swap out packaged components according to your needs. Both Node.JS and Tessel subscribe to modularity. Node has proven out that this approach can scale to production.
+
+In hardware, this is nearly unheard of. It is common to build a first prototype to show technical viability, then throw it out. For production, you start again from scratch to design a PCB that works with parts you can get at the x1000 scale.
+
+With Tessel 2, we say: you shouldn't have to. This concept drove the following design decisions:
+
+* We chose components so that you can use the same ones in production. Every part used for Tessel 2 will continue to be manufactured and supported for a few years. Each can be sourced in low (100x) quantities.
+* Tessel 2 was designed in KiCad. This is perhaps the only a free, open, fully-featured electrical engineering design tool available. Beyond publishing our [design files](https://github.com/tessel/t2-hardware), we want to make sure you can change and build on what we've created.
+* Tessel 2 is physically laid out so that you can easily remove features. A block diagram of Tessel's hardware corresponds to its physical layout, so if you don't need e.g. the ethernet port, you can remove it from the design (and save yourself the cost of the header) in your final product.
+
+![Tessel 2 block diagram](http://softdroid.net/sites/default/files/resize/remote/eb0193ce7bae91d87724d28642c83ab7-500x458.png)
+
+This takes OSHW beyond just licensing. We want you to take our designs and use the parts of them you need in your own products. With your help, we can push the boundaries of OSHW. OSHW should be a platform for serious product development, not only a sharing marketplace for makers.
```

Comment:
Good question! I thought it was in there at the first mention, but now I can't find it. Thank you for catching it!